extkeys/hdkey.go

```diff
@@ -104,16 +104,19 @@ type ExtendedKey struct {
 	CachedPubKeyData []byte // (non-serialized) used for memoization of public key (calculated from a private key)
 }
 
+// MasterSecret is a constant used as hmac key when generating a new master key.
+const MasterSecret = "Bitcoin seed"
```

Comment:
do we need it to be accessible outside `extkeys`?

Reply:
no you are right, we don't need it in general so I'll remove it!